sdks/java/core/src/test/java/org/apache/beam/sdk/io/AvroIOTest.java

```diff
@@ -672,13 +672,13 @@ public void populateDisplayData(DisplayData.Builder builder) {
   }
 
   @Test
-  @Category({ValidatesRunner.class, UsesTestStream.class})
+  @Category({NeedsRunner.class, UsesTestStream.class})
```

Comment:
+1 to this and all tests that are not one of the core primitives. It is true that complex transforms sometimes exercise primitives in exciting ways, but we can and should run integration tests separately from VR which are basically compliance unit tests.

Reply:
I left a few that seemed like they may be exercising per-runner functionality. I agree they probably shouldn't be necessary, but I'm not ready to go through and validate that we have sufficient integration test coverage before removing.